commerce_coordinator/apps/commercetools/filters.py

```diff
@@ -0,0 +1,29 @@
+"""
+Filters used by the commercetools app
+"""
+from openedx_filters.tooling import OpenEdxPublicFilter
+
+
+class OrderHistoryRequested(OpenEdxPublicFilter):
```

Comment:
Thats fine if youd like, i just figured we were eventually going to get rid of FAEcomm, might as well... but i have no preference. I do think they are just copies with different filter type identifier strings.

Reply:
I remember why i did this, the CT Entrypoint (API View) does sorting and such, i didn't want to cross wire too much acrss views... should i just move the sorting from CT View to the FAEcomm API View? Rollback shouldnt be affected but, it would be less performant if its filtering for no reason.